rustls/src/verify.rs

```diff
@@ -591,7 +596,7 @@ impl ClientCertVerifier for AllowAnyAuthenticatedClient {
             &webpki::TlsClientTrustAnchors(&trustroots),
             &chain,
             now,
-            &[], // TODO(@cpu): provide CRLs as appropriate.
+            &[], // TODO(@cpu): parse CRL DER and provide.
```

Comment:
I don't think we should keep this as a separate commit.

Reply:
Collapsed into one commit.